core/misc/jquery-extend-3.4.0.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * For jQuery versions less than 3.4.0, this replaces the jQuery.extend
+ * function with the one from jQuery 3.4.0, slightly modified (documented
+ * below) to be compatible with older jQuery versions and browsers.
+ *
+ * This provides the Object.prototype pollution vulnerability fix to Backdrop
+ * installations running other jQuery versions, including the versions shipped
+ * with Backdrop core and sites using jQuery Update module.
```

Comment:
Do we have a jQuery update module for Backdrop?

Reply:
We do not, but yeah, I reworded it and didn't link to anything in the event it exists in the future.